site/src/pages/tutorials/grpc/blog/implement-delete.mdx

```diff
@@ -0,0 +1,124 @@
+---
+menuTitle: "Implement DELETE"
+order: 6
+category: grpc
+tags:
+  - server
+level: basic
+type: step
+---
+
+# Implementing DELETE operation
+
+So far, we've created, read, and updated a blog post. Now, let's implement 
+and make a call to finally delete a blog post.
+
+<TutorialSteps current={6} />
+
+## What you need
+
+You need to have the files obtained from previous steps:
+
+- [Java code generated](/tutorials/grpc/blog/define-service#compile-proto-file)
+- Main.java
+- BlogService.java
+- BlogClient.java
+
+## 1. Implement service
```

Comment:
(global comment) What do you think of using the noun form instead for section headers?

```suggestion
## 1. Implementing server-side
```

Also, `## 2. Implementing client-side`

Reply:
@jrhee17 ,
I would if this was a guide, but since this is a tutorial, wanted to set the headers as instructions.
Your suggestion is definitely fits the traditional way of technical writing.